index.js

```diff
@@ -8,8 +8,16 @@ module.exports = {
   init: function() {
     this._super.init && this._super.init.apply(this, arguments);
 
-    const reporter = new Reporter();
-    const dependencyChecker = new DependencyChecker(this.project, reporter);
-    dependencyChecker.checkDependencies();
+    // When running `ember <command>`, find the `<command>`
+    const emberPosition = process.argv.findIndex(arg => arg.endsWith('/ember'));
```

Comment:
Will this work on Windows?

Reply:
That is a **great** question, I will investigate!

Reply:
I updated AppVeyor’s configuration to run tests both on Linux _and_ Windows, so we have empirical evidence that yes, [it seems to work on Windows too](https://ci.appveyor.com/project/quaertym/ember-cli-dependency-checker/builds/42608636)! ✨

Reply:
Does cli pass invoked command directly to addons? If so, we can use that instead of parsing it.
Other than that, I am good with the changes.

Reply:
OK, maybe it can be done later on the cli side. I will approve.

Reply:
Let's add GitHub Actions in a separate PR.